_resources/port1.0/fetch/mirror_sites.tcl

```diff
@@ -14,242 +14,204 @@ set portfetch::mirror_sites::sites(afterstep) {
+    http://linorg.usp.br/CPAN/modules/by-module/
+    http://mirrors.linux-bulgaria.org/cpan/modules/by-module/
+    http://mirror.liquidtelecom.com/CPAN/modules/by-module/
+    http://cpan.lnx.sk/modules/by-module/
     http://mirror.low-orbit.net/pub/cpan/modules/by-module/
-    https://mirror.lzu.edu.cn/CPAN/modules/by-module/
+    http://mirrors.m247.ro/CPAN/modules/by-module/
+    http://cpan.melbourneitmirror.net/modules/by-module/
     http://mirror.met.hu/CPAN/modules/by-module/
     http://mirror.metrocast.net/cpan/modules/by-module/
+    http://cpan.mines-albi.fr/modules/by-module/
+    http://mirror-hk.koddos.net/CPAN/modules/by-module/
+    http://mirror.0x.sg/CPAN/modules/by-module/
+    http://cpan.mirror.ac.za/modules/by-module/
+    http://cpan.mirror.ba/modules/by-module/
+    http://cpan.mirror.constant.com/modules/by-module/
+    http://mirror.datacenter.by/pub/CPAN/modules/by-module/
+    http://mirror.ibcp.fr/pub/CPAN/modules/by-module/
+    http://cpan.mirror.iphh.net/modules/by-module/
+    http://mirror.its.dal.ca/cpan/modules/by-module/
+    http://mirror.team-cymru.org/CPAN/modules/by-module/
+    http://mirror.uic.edu/CPAN/modules/by-module/
+    http://mirrors.sonic.net/cpan/modules/by-module/
+    http://www.mirrorservice.org/sites/cpan.perl.org/CPAN/modules/by-module/
+    http://cpan.mmgdesigns.com.ar/modules/by-module/
+    http://mirrors.namecheap.com/CPAN/modules/by-module/
+    http://www.namesdir.com/mirrors/cpan/modules/by-module/
+    http://mirrors.nav.ro/CPAN/modules/by-module/
     http://mirror.navercorp.com/CPAN/modules/by-module/
     http://mirror.nbtelecom.com.br/CPAN/modules/by-module/
-    https://mirror.neostrada.nl/cpan/modules/by-module/
-    https://mirror.netcologne.de/cpan/modules/by-module/
-    https://mirror.nl.leaseweb.net/CPAN/modules/by-module/
+    http://cpan.nctu.edu.tw/modules/by-module/
+    http://mirror.neostrada.nl/cpan/modules/by-module/
+    http://mirror.netcologne.de/cpan/modules/by-module/
+    http://mirrors.neterra.net/CPAN/modules/by-module/
+    http://mirrors.netix.net/CPAN/modules/by-module/
+    http://mirrors.neusoft.edu.cn/cpan/modules/by-module/
+    http://mirrors.nic.cz/CPAN/modules/by-module/
+    http://mirror.nl.leaseweb.net/CPAN/modules/by-module/
+    http://mirrors.nxthost.com/CPAN/modules/by-module/
+    http://mirrors.evowise.com/CPAN/modules/by-module/
     http://mirror.nyi.net/CPAN/modules/by-module/
     http://mirror.optusnet.com.au/CPAN/modules/by-module/
+    http://osl.ugr.es/CPAN/modules/by-module/
+    http://mirror.lzu.edu.cn/CPAN/modules/by-module/
     http://mirror.ox.ac.uk/sites/www.cpan.org/modules/by-module/
-    http://mirror.pop-sc.rnp.br/CPAN/modules/by-module/
-    https://mirror.pregi.net/CPAN/modules/by-module/
-    https://mirror.ps.kz/CPAN/modules/by-module/
-    http://mirror.qnren.qa/CPAN/modules/by-module/
-    https://mirror.rasanegar.com/CPAN/modules/by-module/
+    http://www.perl.com/CPAN/modules/by-module/
```

Comment:
This redirects to https://www.cpan.org/modules/by-module/; we should use that instead.